_docs/ops/runbook/troubleshooting-logsearch.md

```diff
@@ -123,35 +130,101 @@ mutate {
 ```
 
 Edit the `output` section in `/var/vcap/data/ingestor_syslog/tmp/logstash-restore.conf` and change `index` to:
+
 ```
 index => "logs-%{@index_type}-%{[@metadata][index-date]}"
 ```
 
+### Disable the timecop filter
+
+The default Logsearch-for-CloudFoundry configuration will drop any messages older than 24 hours. When reindexing older data this sanity check needs to be disabled.
+
+To disable the timecop filter set the environment variable `TIMECOP_REJECT_LESS_THAN_HOURS` to match the desired rentention policy:
+
+```sh
+export TIMECOP_REJECT_LESS_THAN_HOURS=$((180 * 24))
+```
+
```

Comment:
this is already on this page, isn't it?

Reply:
Yes, it is probably being flagged as new because I moved it, but I will double check!